### PR TITLE
Added API features to generate and parse the JWT token

### PR DESCRIPTION
**Pull request by aschempp**
These are required for the Contao Manager to enable & disable debug mode for Contao 4.8

see https://github.com/contao/contao-manager/commit/690e1f73bf4df7361958d8e1d452f3dc4588a13e

### CODE REVIEW

**Review by leofeyer — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

File: manager-bundle/src/ContaoManager/ApiCommand/ParseJwtCookieCommand.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $payload = $this->jwtManager->parseCookie($input->getArgument('content'));
+
+        $output->write(json_encode($payload));
```
Comment (Toflar) [inline manager-bundle/src/ContaoManager/ApiCommand/ParseJwtCookieCommand.php:57]: Are we sure this is the default on CLI? Sounds like it should be displayed differently and `--format=json` should be an option?

**Review by Toflar — CHANGES_REQUESTED** [review]

File: manager-bundle/src/ContaoManager/ApiCommand/ParseJwtCookieCommand.php (hunk 1 above)
Comment (aschempp) [reply manager-bundle/src/ContaoManager/ApiCommand/ParseJwtCookieCommand.php:57]: The Contao API is always JSON I/O

Comment (Toflar) [reply manager-bundle/src/ContaoManager/ApiCommand/ParseJwtCookieCommand.php:57]: Ah it's only Manager, I see.

**Review by Toflar — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Pull request merged**